sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachineScaleSetOperationsTests.java

```diff
@@ -1504,20 +1606,14 @@ public void canGetOrchestrationType() {
 
         final String vmssName2 = generateRandomResourceName("vmss", 10);
         // create resource through raw method
-        VirtualMachineScaleSetInner result = this
-            .computeManager
-            .virtualMachineScaleSets()
-            .manager()
-            .serviceClient()
-            .getVirtualMachineScaleSets()
-            .createOrUpdate(
-                rgName, vmssName2, options
```

Comment:
the `options` obj can be removed?

Reply:
I've changed direct call from `serviceClient` to premium methods, so options obj are removed.